openjdk/build.xml

```diff
@@ -35,31 +62,36 @@
 				<matches pattern="^[1][01234]" string="${JDK_VERSION}"/>
 			</or>
 			<then>
-				<property name="jtregTar" value="jtreg-4.2.0-tip.tar"/>
+				<property name="jtregTar" value="jtreg_4_2_0_tip.tar"/>
 			</then>
 			<else>
-				<property name="jtregTar" value="jtreg-5.1-b01.tar"/>
+				<property name="jtregTar" value="jtreg_5_1_b01.tar"/>
 			</else>
 		</if>
+		<if>
+			<or>
+				<equals arg1="${JDK_IMPL}" arg2="ibm"  />
+				<equals arg1="${JDK_IMPL}" arg2="openj9" />
+			</or>
+			<then>
+				<antcall target="getDependentLibs"/>
+			</then>
+		</if>
 		<if>
 			<not>
-				<available file="${jtregTar}" />
+				<available file="${LIB_DIR}/${jtregTar}.tar.gz" />
 			</not>
 			<then>
-				<if> <available file="custom_jtreg.tar.gz"/>
+				<if> 
+					<available file="${jtregTar}.tar.gz"/>
```

Comment:
The original logic is checking `custom_jtreg.tar.gz`. We should not change the logic. The same for the line 88-89